lldb/docs/use/mcp.md

```diff
@@ -0,0 +1,68 @@
+# Model Context Protocol (MCP)
+
+LLDB supports for the [Model Context Protocol][1] (MCP). This structured,
+machine-friendly protocol allows AI models to access and interact with external
+tools, such as the debugger. Using MCP, an AI agent can execute LLDB commands
```

Comment:
`such as a debugger` if you're going to generically refer to debuggers. Or `such as LLDB` if you prefer.